pkg/apis/flowcontrol/v1alpha1/defaults.go

```diff
@@ -0,0 +1,29 @@
+/*
+Copyright 2019 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1alpha1
+
+import (
+	"k8s.io/api/flowcontrol/v1alpha1"
+	"k8s.io/kubernetes/pkg/apis/flowcontrol"
+)
+
+// SetDefaults_FlowSchema sets default values for flow schema
+func SetDefaults_FlowSchema(obj *v1alpha1.FlowSchema) {
+	if obj.Spec.MatchingPrecedence == 0 {
```

Comment:
> A matching priority (default value is 1000);

that doesn't comply w/ the KEP. it's said there is a default value for the field. on the other hand, forcing user to think about a matching-precedence will be against user-experience. there should be a default value and admin is supposed to adjust it to resolve conflict among flow-schemas.

Reply:
Oh, you're right, I had forgotten that the KEP said there is a default value for that.  Perhaps I never noticed.  I think it is a bad idea and will lead to bad user experience for the reason I stated.

Reply:
how many flow-schemas can we expect in a cluster? 20~100-ish? it's tough for administrators to think of a unique and reasonable matching-precedence as it grows.. consider how linux defines `niceness` for process.. we don't have to deal with `niceness ` for every process until hitting issues, do we?

Reply:
Since the KEP says there is a default value, I will not object here any more to implementing a default value.  I think it is a bad design, and will argue this matter in a KEP update.